### ISSUE TITLE
Support transaction management in @neo4j/graphql-ogm

### ISSUE THREAD

**Issue by ashlog** [neo4j/graphql#936]
**Is your feature request related to a problem? Please describe.**
Yes, currently, I am doing quite a bit of transactional mutations and would like to keep things atomic. Unfortunately I cannot use ogm since it doesn't have support for transactions, and therefore I am forced to write queries manually with no type hinting.

**Describe the solution you'd like**
Add support for transaction management in `@neo4j/graphql-ogm`.

**Comment by Hosch250** [comment]
Nice to see a PR already open. Would like this to be resolved ASAP, as I'm hitting the exact same issue :).

**Comment by dennisjlee** [comment]
FYI I think this was released in @neo4j/graphql 3.2.0, but #953 was not mentioned in the release notes.

After upgrading @neo4j/graphql and @neo4j/graphql-ogm to the latest (3.2.3), I was able to pass in an open transaction to an OGM operation like this:

```
model.create({
    /* other args here */
    context: {
      executionContext: tx,
    },
  });
```

**Comment by darrellwarde** [comment]
Apologies @dennisjlee, I've now added this to the release notes.

Indeed, this is now implemented, and as such I will close this.
